service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -23,14 +22,17 @@ set -o xtrace
 # user space, such as setting Terra CLI settings which are persisted in the user's $HOME.
 # This post startup script is not run by the same user.
 readonly JUPYTER_USER="jupyter"
+readonly STATUS_ATTRIBUTE="startup_script/status"
```

Comment:
I'd like to suggest:

```
readonly USER_TERRA_CONFIG_DIR=/home/${JUPYTER_USER}/.terra
readonly OUTPUT_FILE="${USER_TERRA_CONFIG_DIR}/post-startup-output.txt"
```

such that the `USER_TERRA_CONFIG_DIR` can be used as needed as new commands are added to this script.

Reply:
👍